packages/inter-protocol/README.md

```diff
@@ -48,12 +48,10 @@ The canonical keys (under `published`) are as follows. Non-terminal nodes could
             - `governance`
             - `vaults`
               - `vault0`
-    - `auction`
+    - `auction` - [snapshot of details](./test/auction/snapshots/test-auctionContract.js.md)
         - `schedule`
         - `governance`
-        - `books`
-          - `book0`
-            - `data`
+        - `book0`
```

Comment:
so we want `book0`, `book1` etc. under `auction` after all?

I'm standing by to see if by "correct this" @turadg meant to change the published structure to include a `book` or `books` node rather than tweaking this README.

Reply:
each commit ends up separately in master so a bugfix commit is reasonable to me to include in any PR that has the appropriate reviewers. if you want to do the work in a different PR up to you.

as an issue to prioritize, https://github.com/Agoric/agoric-sdk/issues/6111 encompasses the requirement. it's prioritized as a blocker for Vaults Release.

Reply:
I tried integrating the change with the existing commit stack, but the changes to the README are in a different commit than where I'd put the code changes. When I tried it, it turned into a mess, because several later commits also edit the same section of the README.  It'll be far simpler as a follow-on.